Configurations/10-main.conf

```diff
@@ -1498,12 +1530,14 @@ my %targets = (
     "mingw" => {
         inherit_from     => [ "BASE_unix", asm("x86_asm"),
                               sub { $disabled{shared} ? () : "x86_uplink" } ],
-        cc               => "gcc",
-        cflags           => picker(default => "-m32 -Wall",
+        CC               => "gcc",
+        CFLAGS           => picker(default => "-Wall",
                                    debug   => "-g -O0",
                                    release => "-O3 -fomit-frame-pointer"),
-        cppflags         => combine("-DL_ENDIAN -DWIN32_LEAN_AND_MEAN",
-                                    "-DUNICODE -D_UNICODE", threads("-D_MT")),
+        cflags           => "-m32",
```

Comment:
cxxflags even here?

Reply:
Well, there is no CXX definition, so there is no contradiction... but if you choose to do something about it, there is similar problem in mingw64.

Reply:
Like you say, there's no CXX definition, and not in mingw64 either...